ketcher-autotests/tests/User-Interface/Clear-Canvas/clear-canvas.spec.ts

```diff
@@ -0,0 +1,106 @@
+import { test } from '@playwright/test';
+import {
+  takeEditorScreenshot,
+  selectTopPanelButton,
+  delay,
+  selectLeftPanelButton,
+} from '@utils/canvas';
+import { clickInTheMiddleOfTheScreen, pressButton } from '@utils/clicks';
+import { openFileAndAddToCanvas } from '@utils/files';
+import { DELAY_IN_SECONDS } from '@utils/globals';
+import {
+  LeftPanelButton,
+  RingButton,
+  TopPanelButton,
+  selectRing,
+} from '@utils/selectors';
+import { addTextBoxToCanvas } from '@utils/selectors/addTextBoxToCanvas';
+
+test.describe('Clear canvas', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
+  });
+  test('Clear Canvas - "Clear canvas" button', async ({ page }) => {
+    // Test case:EPMLSOPKET-1702
+    await selectTopPanelButton(TopPanelButton.Clear, page);
+    await page.getByRole('button', { name: 'Clear Canvas (Ctrl+Del)' }).hover();
+    await delay(DELAY_IN_SECONDS.TWO);
+    await page.getByTitle('Clear Canvas (Ctrl+Del)').click();
```

Comment:
Why do we need this part?

Reply:
Steps in the test: 
1. Move the cursor over the button.
2. Click the 'Clear Canvas' button.